Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 Compiler Features:
  * AST: Export NatSpec comments above each statement as their documentation.
  * Optimizer: Simple inlining when jumping to small blocks that jump again after a few side-effect free opcodes.
+ * Inline Assembly: Do not warn about shadowing by instructions if the shadowed declaration is not a variable.
```

Comment:
Not alphabetically sorted

Reply:
> Inline Assembly: Do not warn about shadowing by instructions if the shadowed declaration is not a variable.

So, iiuc, we do not warn anymore if shadowed declaration is not a variable e.g., a function? Does this mean we retain the warning for shadowed variables in inline assembly? Do we have tests for variables in inline assembly that shadow variables and/or other declaration types e.g., structs, functions etc?

Reply:
I changed it in the meantime, we removed the warning completely.